samples/webforms/embed-a-dashboard-into-an-app/PBIWebApp/Default.aspx

```diff
@@ -0,0 +1,82 @@
+﻿<%@ Page Title="Home Page" Language="C#" MasterPageFile="~/Site.Master" AutoEventWireup="true" CodeBehind="Default.aspx.cs" Inherits="PBIWebApp._Default" %>
+<asp:Content ID="BodyContent" ContentPlaceHolderID="MainContent" runat="server">
+
+    <!-- Sign in -->
+    <div><h1 style="border-bottom:solid; border-bottom-color: silver">Power BI: Integrate dashboard sample web applicaton</h1>
+        <asp:Panel ID="signinPanel" runat="server" Visible="true">     
```

Comment:
let's be consistent with the names.
panels: pnlSometing
buttons: btnSomething
labels: lblSomething
textbox: txtSomething
and so..